api/common/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "dc-api-common"
+version = "0.0.4"
+description = "Common elements to the DC API"
+requires-python = "==3.12.*"
+dependencies = [
+    "httpx[http2]==0.26.0",
+    "mangum==0.17.0",
+    "starlette==0.36.2",
+    "sentry-sdk[starlette]==2.32.0",
+    "polars==1.25.2",
+    "dc-logging-utils"
+]
+
+[tool.uv.sources]
+dc-logging-utils = { git = "https://github.com/DemocracyClub/dc_logging.git", tag = "1.0.2" }
+
+[tool.uv]
+package = true
+
+[tool.uv.build-backend]
```

Comment:
Is this the first toe in the water of UV for packages?

Reply:
No - the second. The dc_logging_utils package is already using it too.

We tried migrating just the "application bit" of that repo to UV and leaving the "package bit" using a setup.py/cfg but the path of least resistance turned out to be migrating the package to use UV.